File: engine/engine-model/src/main/scala/net/server/channel/packet/keymap/AutoMPKeymapChangePacket.scala
package net.server.channel.packet.keymap

class AutoMPKeymapChangePacket(private var _available: Boolean, private var _mode: Int, private var _itemId: Int) extends BaseKeymapChangePacket(_available = _available, _mode = _mode) {
  def itemId: Int = _itemId
}

File: engine/engine-model/src/main/java/net/server/channel/packet/keymap/AutoMPKeymapChangePacket.java
package net.server.channel.packet.keymap;

public class AutoMPKeymapChangePacket extends BaseKeymapChangePacket {
   private final Integer itemId;

   public AutoMPKeymapChangePacket(Boolean available, Integer mode, Integer itemId) {
      super(available, mode);
      this.itemId = itemId;
   }

   public Integer itemId() {
      return itemId;
   }
}
